packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -118,7 +117,19 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
                     this.hostPolicy.enableSingleRequestForShareLinkWithCreate,
             },
             async (event) => {
-                odspResolvedUrl = await createNewFluidFile(
+                // We can delay load this module as this path will not be executed in load flows and create flow
+                // while only happens once in lifetime of a document happens in the background after creation of
+                // detached container.
+                const module = await import(/* webpackChunkName: "createNewModule" */ "./createFile")
```

Comment:
Can you please run it by Loop folks that it's Ok to create a chunk for this scenario? It's possible that we want to have only one chunk combining socket & create file code - not sure

Reply:
At first, I want to put this in a separate bundle and then have a look on the telemetry and see when it is actually getting loaded rather than putting it with socket bundle and then causing it to load the socket bundle too. Later on we can merge the 2 bundles together. @christiango @aboktor

Reply:
Have you done any analysis either on test bundles within Fluid Framework or testing this PR bits against office-bohemia to compare the separate bundle vs re-using the lazy chunk socket code?

Depending on how much overlap in lazy-eligible dependencies there is between the socket and create code, the difference can be non-trivial due to the way webpack split chunk works https://webpack.js.org/plugins/split-chunks-plugin/#splitchunksminsize.

Reply:
So, I did the testing with our bundle size tests and I did not find any difference if I bundled them together or separately. It was a very minimal difference in bytes. So, at least initially we could bundle them separately as then we don't always have to download a big bundle in 1 go.